Julia/src/sw_core.jl

```diff
@@ -0,0 +1,1009 @@
+#=
```

Comment:
Would you mind adding some large-ish comment header blocks above each function to create an easily identifiable separator so readers can easily see where new function starts?  No need for super detailed descriptions in those header blocks, something like what was done for the Fortran version would be sufficient.

Reply:
Sure will! Skipped this for the majority of subroutines.